open_seq2seq/data/speech2text/speech2text.py

```diff
@@ -36,15 +37,23 @@ def get_required_params():
   @staticmethod
   def get_optional_params():
     return dict(DataLayer.get_optional_params(), **{
+        'backend': ['psf', 'librosa'],
         'augmentation': dict,
         'pad_to': int,
         'max_duration': float,
+        'min_duration': float,
         'bpe': bool,
         'autoregressive': bool,
         'syn_enable': bool,
         'syn_subdirs': list,
         'window_size': float,
         'window_stride': float,
+        'dither': float,
+        'norm_per_feature': bool,
+        'window': ['hanning', 'hamming', 'none'],
+        'num_fft': int,
+        'precompute_mel_basis': bool,
+        'sample_freq': int,
```

Comment:
We should add comments explaining what the new parameters do.
Missing 'min_duration', 'dither', 'norm_per_feature', 'window', 'num_fft'

Reply:
dither, norm_per_feature, num_fft are explained in get_speech_features_from_file. We should copy and paste here

Reply:
Fixed